server/services/userService.ts

```diff
@@ -7,10 +9,19 @@ interface UserDetails {
 }
 
 export default class UserService {
-  constructor(private readonly hmppsAuthClient: HmppsAuthClient) {}
+  constructor(
+    private readonly hmppsAuthClient: HmppsAuthClient,
+    private readonly userClientFactory: RestClientBuilder<UserClient>,
+  ) {}
 
   async getUser(token: string): Promise<UserDetails> {
```

Comment:
I'm a bit confused as to which scenarios to call which method in. Is it worth renaming `getUser` to be a bit more descriptive?

Reply:
I've renamed it to `getActingUser`, which I think makes more sense

Reply:
Thanks Stu